walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -28,247 +28,437 @@
 /** */
 @RunWith(JUnit4.class)
 public final class FindMeetingQueryTest {
```

Comment:
Please change the indentation formatting for this file back to 2 spaces, per the Google Java style guidelines.

Reply:
I changed it and looks right in my editor, I am not sure why it's not showing here :(

Reply:
Oooo strange. Can you try reloading your editor?

Reply:
What is the max column size of this file set to? 80?

Reply:
Yeah I had 80. Just added new google java formatter maven plugin. Maybe this will format better.